templates/mypy.sh.tpl

```diff
@@ -27,7 +27,7 @@ main() {
   fi
 
   set +o errexit
-  output=$($mypy {VERBOSE_OPT} --bazel --package-root . --config-file {MYPY_INI_PATH} --cache-map {CACHE_MAP_TRIPLES} -- {SRCS} 2>&1)
+  output=$($mypy {VERBOSE_OPT} --bazel --package-root . --package-root bazel-out/k8-fastbuild/bin --config-file {MYPY_INI_PATH} --cache-map {CACHE_MAP_TRIPLES} -- {SRCS} 2>&1)
```

Comment:
`bazel-out/k8-fastbuild/bin` I don't think that can be hardcoded.

If I do an `ls` on `bazel-out/` I see: 

```
_tmp  darwin-fastbuild	host  stable-status.txt  volatile-status.txt
```

Reply:
Good catch, I didn't know that. I'll look into cross platform methods